src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -340,6 +340,16 @@
     /** Controls the maximum number of index query intersections that will take part in a query */
     SAI_INTERSECTION_CLAUSE_LIMIT("cassandra.sai.intersection.clause.limit", "2"),
 
+    /** Minimum number of reachable leaves for a given node to be eligible for an auxiliary posting list */
+    SAI_MINIMUM_POSTINGS_LEAVES("cassandra.sai.minimum_postings_leaves", "64"),
+
+    /**
+     * Skip, or the sampling interval, for selecting a bkd tree level that is eligible for an auxiliary posting list.
+     * Sampling starts from 0, but bkd tree root node is at level 1. For skip = 4, eligible levels are 4, 8, 12, etc. (no
+     * level 0, because there is no node at level 0).
+     */
+    SAI_POSTINGS_SKIP("cassandra.sai.postings_skip", "3"),
+
```

Comment:
Note: When this rebases on the latest `cep-7-sai`, there might be a few conflicts, but it was just fixing ordering and adding some new properties here.